x/globalfee/querier.go

```diff
@@ -31,7 +31,6 @@ func (g GrpcQuerier) Params(stdCtx context.Context, _ *types.QueryParamsRequest)
 	var maxTotalBypassMinFeeMsgGasUsage uint64
 	ctx := sdk.UnwrapSDKContext(stdCtx)
 
-	// todo: if return err if not exist?
```

Comment:
If this is an invalid state, we should add checks to make it impossible elsewhere

Reply:
The invalid state already has validation:

https://github.com/burnt-labs/xion/blob/3466e9c34a17d3d8c53d8441d9e241892b08573f/x/globalfee/types/genesis.go#L34-L40

It should not be possible to start the module without setting the params.

Reply:
In that case, we can remove the todo, yea?